content/automated-rust-releases.md

```diff
@@ -0,0 +1,582 @@
+
+- _(readme)_ Add link to emacs package support git-cliff ([#307](https://github.com/orhun/git-cliff/issues/307)) - ([fa471c7](https://github.com/orhun/git-cliff/commit/fa471c7178dce184ca6fe5bbb24b9c2db96d68ce))
+```
+
+Great, now we are ready to set up the actual releaser tool!
+
+---
+
+### **release-plz**
+
+> [release-plz](https://release-plz.ieni.dev) creates a fully automated release pipeline, allowing you to easily release changes more frequently, without the fear of doing typos or other subtle manual mistakes you can make when releasing from your terminal.
+
+Unlike `git-cliff`, we don't actually need to install `release-plz` as a command-line tool. It is because `release-plz` runs from the CI (GitHub Actions) and we are not doing anything locally for the releases. However, we still need to tweak some things regarding how we want to release our project.
+
+For configuring `release-plz`, create `release-plz.toml` with the following contents at the root of your repository:
+
+```toml
+[workspace]
+# path of the git-cliff configuration
+changelog_config = "cliff.toml"
+
+# enable changelog updates
+changelog_update = true
+
+# update dependencies with `cargo update`
+dependencies_update = true
+
+# create tags for the releases
+git_tag_enable = true
+
+# disable GitHub releases
+git_release_enable = false
+
+# labels for the release PR
+pr_labels = ["release"]
+
+# disallow updating repositories with uncommitted changes
+allow_dirty = false
+
+# disallow packaging with uncommitted changes
+publish_allow_dirty = false
+
+# disable running `cargo-semver-checks`
+semver_check = false
+```
+
+The important options here are:
+
+- `changelog_config`: should point out to the `git-cliff` configuration which we created in the previous section.
+- `dependencies_update`: setting this to `true` means that you want to run `cargo update` before the each release. Enabling this for updating the transitive dependencies wouldn't hurt since we will already be updating the main dependencies with Dependabot in the future steps.
+- `git_tag_enable`: set this to `true` to create a tag for the new releases.
+- `git_release_enable`: make sure to set this to `false` since we don't want `release-plz` to create a GitHub release for us since that part will be handled by `cargo-dist`.
+
+As for the next step, we should create the _actual_ release automation that is responsible for running `release-plz` for each commit pushed to the repository and creating a "release PR".
+
+**Q**: A release PR?
+
+**A**: Release PR is a pull request that represents the next release and contains the necessary changes such as version bumps and changelog update.
+
+<center>
+
+![release PR](/release-pr.png)
+
+[https://github.com/orhun/daktilo/pull/51](https://github.com/orhun/daktilo/pull/51)
+
+</center>
+
+After the release PR is merged, `release-plz` detects that there is a version change in `Cargo.toml` and runs `cargo publish` to release the new version on crates.io
```

Comment:
```suggestion
After you merge the release PR, `release-plz` detects that there is a version change in `Cargo.toml` and runs `cargo publish` to release the new version on [https://crates.io/](crates.io).
```

Reply:
Specifying the subject ("you" or "the maintainer" in this case), is useful for the reader to understand that merging the release pr is intentional, not part of an automation.